### ISSUE TITLE
VOIP support

### ISSUE THREAD

**Issue by iambeingtracked** [ulyssa/iamb#82]
It would've been cool if iamb supported VOIP calls with turn that Matrix utilizes. VOIP in a TUI client may sound odd, but there's a Tox client with similar [functionality](https://github.com/JFreegman/toxic/blob/master/src/audio_call.c). Not sure that would help, but it would be cool if it were implemented in the future.

**Comment by insanity54** [comment]
I don't think this is an odd request. Now more than ever, there are a bunch of new open-source portable devices coming online which either can't or shouldn't run a full web browser due to memory or graphics processing constraints. Example: SQFMI Beepy, Clockwork Devterm, uConsole, etc. TUI makes a lot of sense for those devices due to the extreme low memory footprint and minimalist, no extras interface. These devices could be a viable android replacement with the addition of phonecalls. That's my dream anyway.
